lib/everypolitician/popolo/person.rb

```diff
@@ -57,6 +58,18 @@ def sort_name
         name
       end
 
+      def name_with_honorifics
+        unless @name.match(/^#{@honorific_prefix}/)
```

Comment:
I think you want to check against `name` (the method) rather than `@name` (the raw data) here. At the moment they're going to be the same, but they're conceptually different things, and if we were to extend the `name` method to do other things, you're almost certainly going to want to include those here.